src/inmanta/server/services/environment_metrics_service.py

```diff
@@ -189,6 +192,36 @@ async def start(self) -> None:
             self.schedule(
                 self.flush_metrics, COLLECTION_INTERVAL_IN_SEC, initial_delay=COLLECTION_INTERVAL_IN_SEC, cancel_on_stop=True
             )
+            # Cleanup metrics once per hour
+            self.schedule(self._cleanup_old_metrics, interval=3600, initial_delay=0, cancel_on_stop=True)
```

Comment:
Shouldn't we enable this always, even if collection has been disabled? I don't have a strong opinion but it would feel slightly more intuitive to me.

Reply:
Fine by my. I was in doubt myself.

Reply:
I changed my mind. the reason why this if-statement exists, is to make sure that the test suite can make sure that the server doesn't act on the metrics tables of the database. So from that perspective it's better to put this one under the if-statement as well. I will add a comment.

Reply:
Ok, I see the reasoning. We'll see if it ever causes trouble. Fine by me to leave it as is for now.

Reply:
Ah but this isn't even an environment setting so it couldn't cause issues anyway.